test/unit/test_artifacts_cli.py

```
@@ -0,0 +1,324 @@
+                self.stdout = out.encode("utf-8")
+
+        if len(args) > 2 and args[1] == "artifact" and args[2] == "ls":
+            return R(ls_output)
+        if len(args) > 2 and args[1] == "artifact" and args[2] == "inspect":
+            return R(inspect_output)
+        raise AssertionError(f"Unexpected command: {args}")
+
+    return fake_run_cmd
+
+    # agent ls json
+    monkeypatch.setattr("ramalama.cli.run_cmd", make_fake_run_cmd("sample-agent"))
```

Ruff (0.16.1)

[error] Undefined name `monkeypatch`

(F821)